kroxylicious-kms-provider-aws-kms-test-support/src/main/java/io/kroxylicious/kms/provider/aws/kms/AbstractAwsKmsTestKmsFacade.java

```diff
@@ -142,6 +147,17 @@ public void deleteKek(String alias) {
         @Override
         public void rotateKek(String alias) {
             var key = read(alias);
+            final ListKeyRotationsRequest listKeyRotationKey = new ListKeyRotationsRequest(key.keyMetadata().keyId());
```

Comment:
Add comments and/or use good variables names to make it clear that
we are using ListKeyRotationsRequest as a probe to discovery the server's capabilities.